ion-schema/src/violation.rs

```diff
@@ -210,10 +210,9 @@ macro_rules! assert_equivalent_violations {
     };
 }
 
-#[macro_export]
-
 /// Equivalence for `Violation`s is not supported due to its tree structure of having children violations.
 /// This macro can be used for comparing if two violations are not equal and uses `flattened_violations` for the comparison.
+#[macro_export]
```

Comment:
🗺️ There shouldn't be empty lines between an attribute and the thing to which it applies.